nss-cohort-36/bangazon-client-imagination-station-react
# added string validation to product create form

# Description
Added check for !@#$%^&*() characters in product create form.

Given a user fills out the product details form
When the user clicks the affordance to submit the product
And either the title, or description, has any of the following characters in it !@#$%^&*()
Then the user should be informed that the corresponding field must be updated without those characters


Fixes #30
## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. `git fetch --all`
2. `git checkout lr-special-characters-check-30`
3. Go to the product create form and make sure you cannot enter and submit any of these characters !@#$%^&*()  in the title and description. You should receive and alert on submit. Make sure you can still create a new product after you take the characters out.

# Checklist:
